### PR TITLE
Misleading documentation of includeDependencyManagement

### PR DESCRIPTION
**Pull request by rotty3000**

### CODE REVIEW

**Comment by rotty3000** [comment]
@cdelg what do you think?

**Comment by cdelg** [comment]
Much clearer, thanks!

File: maven/bnd-resolver-maven-plugin/README.md [hunk 1]
```diff
@@ -57,16 +57,20 @@ The `-runee` and `-runrequires` values can be inferred from the maven project as
...
+|`useMavenDependencies`       | If `true`, adds the project dependencies subject to `scopes` to the collection of files to include in the *implicit repository*. _Defaults to `true`._|
 |`reportOptional`             | If `true`, resolution failure reports will include optional requirements. _Defaults to `true`._|
 |`scopes`                     | Specify from which scopes to collect dependencies. _Defaults to `compile, runtime`._ Override with property `bnd.resolve.scopes`.|
-|`includeDependencyManagement`| Include `<dependencyManagement>` when locating bundles during the bndrun resolution. _Defaults to `false`._ Override with property `bnd.resolve.include.dependency.management`.|
+|`includeDependencyManagement`| Include `<dependencyManagement>`subject to `scopes` when collecting files to include in the *implicit repository*. _Defaults to `false`._ Override with property `bnd.resolve.include.dependency.management`.|
```
Comment (bjhargrave) [inline maven/bnd-resolver-maven-plugin/README.md]: MIssing a space after `Include '<dependencyManagement>'`

File: maven/bnd-run-maven-plugin/README.md [hunk 2]
```diff
@@ -115,12 +115,16 @@ The `-runee` and `-runrequires` values can be inferred from the maven project as
...
+|`useMavenDependencies`       | If `true`, adds the project dependencies subject to `scopes` to the collection of files to include in the *implicit repository*. _Defaults to `true`._|
 |`scopes`                     | Specify from which scopes to collect dependencies. _Defaults to `compile, runtime`._ Override with property `bnd.run.scopes`.|
-|`includeDependencyManagement`| Include `<dependencyManagement>` when locating bundles during the bndrun resolution. _Defaults to `false`._ Override with property `bnd.run.include.dependency.management`.|
+|`includeDependencyManagement`| Include `<dependencyManagement>`subject to `scopes` when collecting files to include in the *implicit repository*. _Defaults to `false`._ Override with property `bnd.run.include.dependency.management`.|
```
Comment (bjhargrave) [inline maven/bnd-run-maven-plugin/README.md]: MIssing a space after `Include '<dependencyManagement>'`

File: maven/bnd-testing-maven-plugin/README.md [hunk 3]
```diff
@@ -66,8 +70,8 @@ The `-runee` and `-runrequires` values can be inferred from the maven project as
...
+|`useMavenDependencies`          | If `true`, adds the project dependencies subject to `scopes` to the collection of files to include in the *implicit repository*. _Defaults to `true`._|
 |`reportOptional`                | If `true`, resolution failure reports (see `resolve`) will include optional requirements. _Defaults to `true`._|
 |`scopes`                        | Specify from which scopes to collect dependencies. _Defaults to `compile, runtime`._ Override with property `bnd.testing.scopes`.|
-|`includeDependencyManagement`   | Include `<dependencyManagement>` when locating bundles during the bndrun resolution. _Defaults to `false`._ Override with property `bnd.testing.include.dependency.management`.|
+|`includeDependencyManagement`   | Include `<dependencyManagement>`subject to `scopes` when collecting files to include in the *implicit repository*. _Defaults to `false`._ Override with property `bnd.testing.include.dependency.management`.|
```
Comment (bjhargrave) [inline maven/bnd-testing-maven-plugin/README.md]: MIssing a space after `Include '<dependencyManagement>'`

**Pull request merged**